src/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do-list</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;1,400&display=swap" rel="stylesheet">
+<script src="https://kit.fontawesome.com/8eb467c874.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="containar">
+        <h1>Today's To DO</h1>
+        <input class="addlist" id="input" placeholder="Add to your list..." type="text">
+    <ul id="mission" class="mission"></ul>
```

Comment:
It would be better to give meaningful names, for easy code reading.

Reply:
what name do you main in particular

Reply:
The name for the Unordered list 'mission' sounds off-topic.

Reply:
not at all what is a task if it's now not a mission that you must complete